### ISSUE TITLE
Integration with Contacts app

### ISSUE THREAD

**Issue by knobtviker** [tgalal/wazapp#52]
Contacts are saved in users tracker database. Everything inside is connected using Nepomuk. Wazapps .service file has to be altered mildly but Wazapp has to store contacts (or actually flag them as users) in tracker database.
Once a new IM has been added to every contact matched, a button will appear with Wazapp icon and default send-text-balloon. When clicked on a baloon a handler will start Wazapp with open conversation (very similar to Forward function), but when clicked on a Wazapp pushname displayed (do mind it's not the actual username) the latest status or message can be displayed.
This can be achieved via DBUS or handlers in service file, but only once the database is merged into main tracker database.
